ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/RemoveUnusedOpensFix.fs

```diff
@@ -29,14 +29,5 @@ type RemoveUnusedOpensFix(warning: UnusedOpenWarning) =
                 | null -> ()
                 | warning ->
 
-                let openStatement = warning.OpenStatement
-
-                let first = getFirstMatchingNodeBefore isInlineSpaceOrComment openStatement
-                let last =
-                    openStatement
-                    |> skipSemicolonsAndWhiteSpacesAfter
-                    |> getThisOrNextNewLine
-
-                deleteChildRange first last
-
+                OpensUtil.removeUnusedOpen warning.OpenStatement
```

Comment:
Wouldn't it be better to place it to `| warning ->` line? It seems short enough.